otter/integration/tests/test_convergence.py

```diff
@@ -321,20 +321,93 @@ def test_scale_down_after_oobd_non_constrained(self):
             Delete z (where z<x) of the servers out of band
             Scale down by (y) servers (where z + y < x)
             Validate end state of (N + x - y) servers for:
+                - z < |y|
+
         """
+        rcs = TestResources()
 
-        min_servers = 2
-        set_to_servers = 7
-        oobd_servers = 2
-        scale_servers = -3
-        # This only applies if not constrained by max/min
-        converged_servers = set_to_servers + scale_servers
+        N = 2
+        x = 7
+        z = 2
+        y = -3  # Scale down 3 servers
+
+        return self._scale_down_after_oobd_non_constrained_param(
+            rcs, min_servers=N, set_to_servers=x, oobd_servers=z,
+            scale_servers=y)
+
+    test_scale_down_after_oobd_non_constrained_z_lessthan_y.timeout = 1800
+
+    def test_scale_down_after_oobd_non_constrained_z_greaterthan_y(self):
+        """
+        Validate the following edge case:
+        - When scaling down after an out of band delete (OOBD) that is
+          not constrained by the group max or min, the group stabilizes at a
+          number of servers consistent with scaling from the active capacity
+          before the OOBD. (i.e. The final result should be as if the OOBD
+          never happened.)
+
+            Create a group with min N servers
+            Scale group to (N + x) servers
+            Delete z (where z<x) of the servers out of band
+            Scale down by (y) servers (where z + y < x)
+            Validate end state of (N + x - y) servers for:
+                - z > |y|
+
+        """
+
+        rcs = TestResources()
+
+        N = 2
+        x = 7
+        z = 3
+        y = -2  # Scale down 3 servers
 
+        return self._scale_down_after_oobd_non_constrained_param(
+            rcs, min_servers=N, set_to_servers=x, oobd_servers=z,
+            scale_servers=y)
+
+    test_scale_down_after_oobd_non_constrained_z_greaterthan_y.timeout = 1800
+
+    def test_scale_down_after_oobd_non_constrained_z_equal_y(self):
+        """
+        Validate the following edge case:
+        - When scaling down after an out of band delete (OOBD) that is
+          not constrained by the group max or min, the group stabilizes at a
+          number of servers consistent with scaling from the active capacity
+          before the OOBD. (i.e. The final result should be as if the OOBD
+          never happened.)
+
+            Create a group with min N servers
+            Scale group to (N + x) servers
+            Delete z (where z<x) of the servers out of band
+            Scale down by (y) servers (where z + y < x)
+            Validate end state of (N + x - y) servers for:
+                - z == |y|
+
+        """
         rcs = TestResources()
 
+        N = 2
+        x = 7
+        z = 3
+        y = -3  # Scale down 3 servers
+
+        return self._scale_down_after_oobd_non_constrained_param(
+            rcs, min_servers=N, set_to_servers=x, oobd_servers=z,
```

Comment:
Nitpick on terminology and variable naming - if we are setting desired capacity to x, then shouldn't the docstring be:

`Scale group to x servers (where x > N)`

instead of 

`Scale group to (N + x) servers`